docs/v2-system-addon/data_dictionary.md

```diff
@@ -183,7 +183,8 @@ Schema definitions/validations that can be used for tests can be found in `syste
 | `action` | [Required] Either `activity_stream_event`, `activity_stream_session`, or `activity_stream_performance`. | :one:
 | `addon_version` | [Required] Firefox build ID, i.e. `Services.appinfo.appBuildID`. | :one:
 | `client_id` | [Required] An identifier for this client. | :one:
-| `card_type` | [Optional] ("bookmark", "pocket", "trending", "pinned") | :one:
+| `card_type` | [Optional] ("bookmark", "pocket", "trending", "pinned", "search") | :one:
+| `search_vendor` | [Optional] the vendor of the search shortcut, one of ("google", "amazon", "wikipedia", "duckduckgo", "bing", etc.). This field only exists when `card_type = "search"` | :one:
```

Comment:
I might be misreading the code in the previous file, but wouldn't `this.props.link.hostname` return `google.com`, `amazon.com` or `wikipedia.org`?

Reply:
I believe `hostname` is already shortened by [shortURL](https://github.com/mozilla/activity-stream/blob/0cf21c41e96d81cfa143b7da96a9959f8e4c4abf/lib/ShortURL.jsm#L38)